docs/source/conf.py

```diff
@@ -24,8 +24,10 @@
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = ['sphinx.ext.autodoc',
-              'sphinx.ext.mathjax', 'sphinx.ext.intersphinx',
+              'sphinx.ext.mathjax',
+              'sphinx.ext.intersphinx',
               'sphinx.ext.viewcode',
+              'sphinx.ext.graphviz',
```

Comment:
Is this an extra sphinx package that `firedrake-install --documentation-dependencies` needs to fetch?

Reply:
I just tried it in a docker image and I think the answer is no. But it does depend on having graphviz, which is a brew / apt install not a pip install. I think I want to do something like

```
if args.documentation_dependencies:
    required_packages.append("graphviz")
```

in the install script -- does that make sense?